test/string.js

```diff
@@ -3180,6 +3180,42 @@ describe('string', () => {
             ], done);
         });
 
+        it('validates an base64 string with explicit padding required', (done) => {
```

Comment:
Could we add some tests around passing in the `padding` parameter so that we are making sure that it requires a boolean and other  inputs are rejected.